usmqe_tests/api/gluster/conftest.py

```diff
@@ -3,22 +3,13 @@
 from usmqe.gluster import gluster
 import usmqe.inventory as inventory
 
-
-@pytest.fixture
-def valid_cluster_id(valid_session_credentials):
-    """
-    Generate valid id of imported cluster.
-    """
-    # TODO change
-    api = glusterapi.TendrlApiGluster(auth=valid_session_credentials)
-    cluster_list = api.get_cluster_list()
-    return cluster_list[0]["cluster_id"]
```

Comment:
If I redo all testcases it will be huge PR. My intention was to have remade only basic testcase so we can build upon that change and redo other testcases. ```invalid_cluster_id``` fixture is still valid because it is used during negative testing.